core/lib.deno_core.d.ts

```diff
@@ -141,5 +141,28 @@ declare namespace Deno {
     ): undefined | UncaughtExceptionCallback;
 
     export type UncaughtExceptionCallback = (err: any) => void;
+
+    /**
+     * Enables collection of stack traces of all async ops. This allows for
+     * debugging of where a given async op was started. Deno CLI uses this for
+     * improving error message in op sanitizer errors for `deno test`.
+     *
+     * **NOTE:** enabling tracing has a significant negative performance impact.
+     * To get high level metrics on async ops with no added performance cost,
+     * use `Deno.core.metrics()`.
+     */
+    function enableOpCallTracing(): void;
```

Comment:
why not just have a boolean value that can be assigned, rather than a function:
```
Deno.core.enableOpCallTracing = true;
```
This allows for future flexibility (if you want to turn it off) and less wrapper.

Reply:
The chosen approach is actually intentional: the "enabled" flag is a boolean (a primitive), so assigning it to `Deno.core` assigns it by value, not by reference. To fix this, we'd need to change the check in `opAsync` to be `if (core.opCallTracingEnabled)` instead of `if (opCallTracingEnabled)`. The former introduces an additional property lookup to all `opAsync` calls which I don't really want as it is a super hot path.